stitching/cli/stitch.py

```diff
@@ -263,6 +263,13 @@ def create_parser():
         choices=Timelapser.TIMELAPSE_CHOICES,
         type=str,
     )
+    parser.add_argument(
```

Comment:
I would like to call it output_params and put it after `--output` to have a closer connection here. 

I would like to extend the help a bit. Maybe just describe your example, how cv.IMWRITE_TIFF_COMPRESSION, cv.IMWRITE_TIFF_COMPRESSION_NONE leads to 259 1 (the integers to be used in the CLI)